tembo-operator/tests/integration_tests.rs

```diff
@@ -2168,7 +2168,7 @@ mod test {
             .as_ref()
             .expect("Annotations should be present");
         let basedomain = std::env::var("DATA_PLANE_BASEDOMAIN").unwrap();
-        let expected_hostname = format!("{}.{}", namespace, basedomain);
+        let expected_hostname = format!("dedicated.{}.{}", namespace, basedomain);
```

Comment:
Let's add one for when `is_standby == true`, where `expected_hostname` would equal `format!("dedicated-ro.{}.{}", namespace, basedomain)`